kernel-module/xt_RTPENGINE.c

```diff
@@ -195,29 +195,29 @@ struct rtpengine_output;
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3,10,0)
 static kuid_t proc_kuid;
 static uint proc_uid = 0;
-module_param(proc_uid, uint, 0);
+module_param(proc_uid, uint, S_IRUGO | S_IWUSR);
```

Comment:
The modparams for the /proc entries are only used during module load, so there isn't really any point in having them writable (unless you add your own setter function)

Reply:
Agree and actually do not see currently a reason to make parameters writable, only `log_errors` make sense to be writeable.